### ISSUE TITLE
Cancel order item without refunding

### ISSUE THREAD

**Issue by chladog** [vendure-ecommerce/vendure#569]
**Is your feature request related to a problem? Please describe.**
We can now cancel an item from the order, but only with refund functionality. If we adjust refund amount to sum 0, then the refund button is disabled.

**Describe the solution you'd like**
It's possible to agree on different fulfillment by direct communication with the customer - then we just want to cancel the item from the order, but we do not want to refund him any amount.

**Comment by michaelbromley** [comment]
Hi, can you help me to understand the use-case a bit better? 

If you are cancelling and OrderItem but not refunding, what then happens? Are you sending a different item instead?

**Comment by chladog** [comment]
Yes, for example if we find out error in stock and we do not have it yet it was already ordered by customer we can agree with the customer on sending product with similar price instead and just cancel that particular item from the order without refunding the money. - this is now unpossible - either we have to go to that missing product and raise stock so we can _fulfill_ the order, or _cancel **&** refund_.

The core of the problem is lack of feature #314, until then we depend on direct communication with customers and cancelling items / adding products "out of the system"._

**Comment by michaelbromley** [comment]
Ok, thanks for the info. It _is_ possible to cancel items without a refund via the Admin API. The current issue is more a limitation of the Admin UI. I think a smarter UI layout can solve this. Something like:

item | SKU | quantity | select | cancel |  refund
-----|------|----------|-----------|------|--------
widgetA | 01000 | 1 | [1] | [✅] | [ ]
